tests/dat/actions/countdown.js

```diff
@@ -2,17 +2,19 @@
  * An action that invokes itself recursively, programmatically using the whisk
  * Javascript API.
  */
+var openwhisk = require('openwhisk')
+
 function main(params) {
+    var wsk = openwhisk({ignore_certs: true})
+
     var n = parseInt(params.n);
     console.log(n);
     if (n === 0) {
         console.log('Happy New Year!');
     } else if (n > 0) {
-        return whisk.invoke({
-            name : 'countdown',
-            parameters : {
-                n : n - 1
-            }
+        return wsk.actions.invoke({
+            actionName: process.env['__OW_ACTION_NAME'].split('/')[2],
```

Comment:
Agree with keep things consistent. Maybe we should do this ASAP, since it will be hard to make backward-incompatible changes in the `npm` `openwhisk` package once it's been officially documented that this is how to use it?